src/python/WMCore/Wrappers/JsonWrapper/JSONThunker.py

```diff
@@ -262,7 +262,7 @@ def _unthunk(self, jsondata):
         """
         _unthunk - does the actual work for unthunk
         """
-        if (type(jsondata) == types.UnicodeType):
+        if (type(jsondata) == str):
```

Comment:
I'm responsible for this catastrophe of thunk/untunk code (sorry). Assuming we want to move from python 2.X to 3.X, this needs to be changed to isinstance() to handle unicode properly